src/main/java/com/google/cloud/tools/appengine/configuration/AppYamlProjectStageConfiguration.java

```diff
@@ -85,7 +85,7 @@ public static Builder builder() {
   /**
    * Create a new builder for AppYamlProjectStageConfiguration.
    *
-   * @deprecated Use newBuilder().appEngineDirectory(appEngineDirectory).artifact(artifact)
+   * @deprecated Use builder().appEngineDirectory(appEngineDirectory).artifact(artifact)
```

Comment:
I think this doc was right. This method is marked `@Deprecated`.

Reply:
Hrmm, I think this change is required. There is no "newBuilder", just the non-deprecated "builder" at:  https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/842/files/55cd842928e906a30b3252ecbb026308753248af#diff-2f220793beb46c697d27366fe2a5a214L81

Reply:
Ahhh, you're right.

Reply:
Ah, you're right.